boards/px4/sitl/default.px4board

```diff
@@ -43,7 +44,6 @@ CONFIG_MODULES_SIMULATOR=y
 CONFIG_MODULES_TEMPERATURE_COMPENSATION=y
 CONFIG_MODULES_UUV_ATT_CONTROL=y
 CONFIG_MODULES_UUV_POS_CONTROL=y
-CONFIG_MODULES_GIMBAL=y
```

Comment:
Unnessary change? The module was readded above

Reply:
I'm pretty sure that I didn't make this change.

Reply:
Then all the more reasons to remove it from the PR 😄